### ISSUE TITLE
Test for WCAG failure F69: Resizing visually rendered text up to 200 percent causes the text, image or controls to be clipped, truncated or obscured

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#222]
http://www.w3.org/WAI/GL/2013/WD-WCAG20-TECHS-20130711/F69

The objective of this failure condition is to describe a problem that occurs when changing the size of text causes text to be clipped, truncated, or obscured, so that it is no longer available to the user. In general, this failure occurs when there is no way for a user agent's layout engine to honor all the layout hints in the HTML at the new font size. Some of the ways in which this can occur include:
[ ] Setting the overflow property of the enclosing element to hidden
[ ] Using absolutely positioned content
[ ] Creating popups that aren't big enough for their content at the new font size

Note: The Working Group has discovered many misunderstandings about how to test this failure. We are planning to revise this failure in a future update. Until then, if the content passes the success criterion using any of the listed sufficient techniques, then it does not meet this failure.
## QUAIL

This failure is currently in revision. So we could better wait until we have more test information?
